Modules/xSharePoint/DSCResources/MSFT_xSPFarmAdministrators/MSFT_xSPFarmAdministrators.psm1

```diff
@@ -0,0 +1,224 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)] [System.String] $Name,
+        [parameter(Mandatory = $false)] [System.String[]] $Members,
+        [parameter(Mandatory = $false)] [System.String[]] $MembersToInclude,
```

Comment:
MembersToInclude & MembersToExclude are unneccessary to be included as with DSC the LCM will update this on the farm after a new version of the Configuration is Pushed (or pulled) to the servers in the farm.

The reasoning behind as referenced in the other PR would be valid if this was just a custom PS Script that you were running to maintain the configuration but with DSC you should just be updating the minimum required to the configuration and then request to "Make it so" as Jeffery Snover would say